src/ape/managers/networks.py

```diff
@@ -31,12 +32,9 @@ def ecosystems(self) -> Dict[str, EcosystemAPI]:
                 data_folder=self.config.DATA_FOLDER / plugin_name,
                 request_header=self.config.REQUEST_HEADER,
```

Comment:
```suggestion
                request_header=json.dumps(self.config.REQUEST_HEADER),
```

(will require importing `json` first.